### ISSUE TITLE
snap: Desktop file didn't specify Exec field

### ISSUE THREAD

**Issue by christian-rauch** [cnr-isti-vclab/meshlab#367]
I installed MeshLab from the snapstore on Ubuntu 18.04.

When lunching MeshLab via the desktop menu entry, I get the error message>
`Desktop file didn't specify Exec field`:
![meshlab_exec_field](https://user-images.githubusercontent.com/8226248/46290852-dbdf4e80-c584-11e8-8986-a0cfa91c8600.png)

Running `meshlab` via command line works.

**Comment by HTLife** [comment]
@christian-rauch  This is caused by the improper setting of .desktop file.
You could fix this by adding `Exec=/snap/bin/meshlab` to the tail of `/var/lib/snapd/desktop/applications/meshlab_meshlab.desktop`.

**Comment by christian-rauch** [comment]
It would be nice if this could be fixed upstream so it is fixed for all users and stays persistent after updates.

**Comment by askkvn** [comment]
> @christian-rauch This is caused by the improper setting of .desktop file.
> You could fix this by adding `Exec=/snap/bin/meshlab` to the tail of `/var/lib/snapd/desktop/applications/meshlab_meshlab.desktop`.

Thanks for workaround.:)

**Comment by christian-rauch** [comment]
I've noticed that there are two additional starters:
```
$ locate meshlab.desktop
/snap/meshlab/4/meta/gui/meshlab.desktop
/snap/meshlab/4/snap/gui/meshlab.desktop
/var/lib/snapd/desktop/applications/meshlab_meshlab.desktop
```

The file `meshlab.desktop` actually contains the `Exec=` field as in https://raw.githubusercontent.com/cnr-isti-vclab/meshlab/master/src/install/snap201612/snap/gui/meshlab.desktop. How does the starter `meshlab_meshlab.desktop` get installed? Maybe it would be sufficient, to remove it from the installation procedure.

**Comment by canonex** [comment]
I confirm the bug.
I installed Meshroom today and I have this problem.

**Comment by alemuntoni** [comment]
This issue should be solved in the latest version (2020.06). Feel free to reopen if the problem persists.
